helm/cluster/files/helmreleases/node-problem-detector.yaml

```
@@ -0,0 +1,15 @@
+appName: node-problem-detector
+configKey: nodeProblemDetector
+catalog: cluster
```

Shouldn't this be the `default` catalog, because it's a default app? Or is it running on the MC like all the Crossplane resources apps?

---

This field isn't used with Releases, so I didn't test it. Fixing it.

```suggestion
catalog: default
```